projects/demo/src/modules/markup/spaces/examples/1/index.html

```diff
@@ -1,59 +1,137 @@
 <h2 i18n class="title">Margin top <code>tui-space_top-&lt;value&gt;</code></h2>
 <div class="row">
-    <div class="example tui-space_top-1 tui-space_right-2">tui-space_top-1</div>
-    <div class="example tui-space_top-2 tui-space_right-2">tui-space_top-2</div>
-    <div class="example tui-space_top-3 tui-space_right-2">tui-space_top-3</div>
-    <div class="example tui-space_top-4 tui-space_right-2">tui-space_top-4</div>
-    <div class="example tui-space_top-5 tui-space_right-2">tui-space_top-5</div>
+    <div class="example tui-space_top-1 tui-space_right-2">
+        <tui-doc-copy [cdkCopyToClipboard]="'tui-space_top-1'">
```

Comment:
Hey 👋
Unfortunately, there is no linter rules or public code style guideline for this (we'll think about it), but we prefer to write `cdkCopyToClipboard="tui-space_top-1"` passing a string

Reply:
Haha I prefer it as well and had written it before but changed it to match the template of the typography demo 🙈

Reply:
@MarsiBarsi fixed for spaces and typo docs 👍

Reply:
oh, it seems to be a mistake by us, thank you